library/std/src/sys/windows/fs.rs

```diff
@@ -47,6 +47,9 @@ pub struct ReadDir {
     first: Option<c::WIN32_FIND_DATAW>,
 }
 
+type AlignedReparseBuf =
+    AlignedAs<c::REPARSE_DATA_BUFFER, [u8; c::MAXIMUM_REPARSE_DATA_BUFFER_SIZE]>;
```

Comment:
I'll do a followup that makes this use `MaybeUninit<u8>`. I was going to do that first, but then noticed that this code was wrong.